aws/scenarios/vm/os/os.go

```diff
@@ -12,15 +12,6 @@ type OS interface {
 	GetTenancy() string
 }
 
-func GetOS(env aws.Environment, osType os.OSType) OS {
-	switch osType {
-	case os.WindowsOS:
-		return newWindows(env)
-	case os.UbuntuOS:
-		return newUbuntu(env)
-	case os.MacosOS:
-		return newMacOS(env)
-	default:
-		panic("OS not supported")
-	}
+func GetOSes(env aws.Environment) []OS {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
Could this be named `GetSupportedOSes` ?

Reply:
I didn't change the name of this function in this PR but `GetSupportedOSes` is more relevant.

Reply:
Code updated.